service-tests/php-v-packagist.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'php', title: 'PHP version from Packagist' });
+module.exports = t;
+
+t.create('gets the package version of symfony')
+  .get('/v/packagist/symfony/symfony/dev-master.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'PHP', value: '^7.1.3' }));
```

Comment:
This test is too brittle, since it'll break when this changes. Could you make a regex for this instead? Maybe you could adapt [this one](https://github.com/badges/shields/blob/632c70b6b516389890d43869c26278fa355b5c01/service-tests/helpers/validators.js#L12).